Extension/src/LanguageServer/extension.ts

```diff
@@ -262,8 +262,13 @@ async function installVsix(vsixLocation: string, updateChannel: string): Promise
 
         // Install the VSIX
         return new Promise<void>((resolve, reject) => {
-            let process: ChildProcess = spawn(vsCodeScriptPath, ['--install-extension', vsixLocation]);
-            if (process.pid === undefined) {
+            let process: ChildProcess;
+            try {
+                process = spawn(vsCodeScriptPath, ['--install-extension', vsixLocation], {shell: true});
```

Comment:
Does turning on `shell` result in a terminal/cmd window opening up?  I would think that the real problem here is simply that the `vsCodeScriptPath` has quotes in it and we just need to remove them.

Reply:
I just tested this on Windows and removing the quotes from `vsCodeScriptPath` fixes the spawn issue.  It should also fix it for macOS.